client/src/dojo/modelManager/TileManager.ts

```
@@ -277,21 +325,28 @@
-          ? new CairoOption<Number>(CairoOptionVariant.Some, resourceType)
-          : new CairoOption<Number>(CairoOptionVariant.None, 0),
-    });
+    try {
+      await this.setup.systemCalls.create_building({
+        signer: useAccountStore.getState().account!,
+        entity_id: entityId,
+        directions: directions,
+        building_category: buildingType,
+        produce_resource_type:
+          buildingType == BuildingType.Resource && resourceType
+            ? new CairoOption<Number>(CairoOptionVariant.Some, resourceType)
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)